components/Home/Hero/__tests__/index.test.tsx

```diff
@@ -0,0 +1,109 @@
+import { render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import Hero from '..';
+import type { NodeReleaseData } from '../../../../types';
+
+const mockNodeReleaseData: NodeReleaseData[] = [
```

Comment:
We should create fixture I thinks it's will ship with [this](https://github.com/nodejs/nodejs.org/pull/5365#discussion_r1187438453)

Reply:
I believe that mocking like this should be sufficient for this test file.

Reply:
Make sense

Reply:
As the fixture is already in progress, Is there any action item for me?

Reply:
Leave as is and update when the fixture is available.

Reply:
@HinataKah0's PR is almost ready so, we should be able to very soon use their fixtures!